apps/builder/app/builder/features/style-panel/sections/backdrop-filter/backdrop-filter.tsx

```diff
@@ -0,0 +1,112 @@
+import { getStyleSource } from "../../shared/style-info";
+import { getDots } from "../../shared/collapsible-section";
+import { InfoCircleIcon, PlusIcon } from "@webstudio-is/icons";
+import { addLayer } from "../../style-layer-utils";
+import { parseFilter } from "@webstudio-is/css-data";
+import { LayersList } from "../../style-layers-list";
+import { FilterLayer } from "../filter/filter-layer";
+
+export const properties = ["backdropFilter"] satisfies Array<StyleProperty>;
+
+const property: StyleProperty = properties[0];
+const label = "Backdrop Filters";
+const INITIAL_BACKDROP_FILTER = "blur(0px)";
+
+export const Section = (props: SectionProps) => {
+  const { currentStyle, deleteProperty } = props;
+  const [isOpen, setIsOpen] = useState(false);
+  const layerStyleSource = getStyleSource(currentStyle[property]);
+  const value = currentStyle[property]?.value;
+
+  return (
+    <CollapsibleSectionRoot
+      fullWidth
+      label={label}
+      isOpen={isOpen}
+      onOpenChange={setIsOpen}
+      trigger={
+        <SectionTitle
+          dots={getDots(currentStyle, properties)}
+          suffix={
+            <Tooltip content={"Add a backdrop-filter"}>
+              <SectionTitleButton
+                prefix={<PlusIcon />}
+                onClick={() => {
+                  addLayer(
+                    property,
+                    parseFilter(INITIAL_BACKDROP_FILTER),
+                    currentStyle,
+                    props.createBatchUpdate
+                  );
+                  setIsOpen(true);
+                }}
+              />
+            </Tooltip>
+          }
+        >
+          <PropertyName
+            title={label}
+            style={currentStyle}
+            properties={properties}
+            description="Backdrop filters are similar to filters, but are applied to the area behind an element. This can be useful for creating frosted glass effects."
+            label={
+              <SectionTitleLabel color={layerStyleSource}>
+                {label}
+              </SectionTitleLabel>
+            }
+            onReset={() => deleteProperty(property)}
+          />
+        </SectionTitle>
+      }
+    >
+      {value?.type === "tuple" && value.value.length > 0 && (
+        <LayersList<FunctionValue, TupleValue>
+          {...props}
+          property={property}
+          layers={value}
+          renderLayer={(layerProps) => (
+            <FilterLayer
+              {...layerProps}
+              key={layerProps.index}
+              label={label}
+              tooltip={
+                <Tooltip
+                  content={
+                    <Flex gap="2" direction="column">
+                      <Text variant="regularBold">{label}</Text>
+                      <Text variant="monoBold">backdrop-filter</Text>
+                      <Text>
+                        Applies graphical effects like
+                        <br />
```

Comment:
did you need to add br?  wondering why

Reply:
Nope, Just took it out of the content component and pasted here. It was a long standing thing, if we are not using `<br />` it is not preserving the new lines when the tooltip content is rendered. 
https://github.com/webstudio-is/webstudio/blob/main/apps/builder/app/builder/features/style-panel/sections/filter/filter-content.tsx#L76-L86

Reply:
what does it look like without those breaks?

Reply:
with `<br />`
<img width="319" alt="Screenshot 2024-05-07 at 14 10 04" src="https://github.com/webstudio-is/webstudio/assets/11075561/cb37477e-2386-4df1-a593-a2af2134881e">

without `<br />`
<img width="398" alt="Screenshot 2024-05-07 at 14 09 53" src="https://github.com/webstudio-is/webstudio/assets/11075561/844bd04e-c5ab-4e9e-82da-ecbab4bab2da">

Reply:
Ohh this should be handled by enforcing the maximum tooltip width and not with br